# Publish all joint states -----------------------------------
#joint_state_controller:
#  type: joint_state_controller/JointStateController
#  publish_rate: 50
#------------------head actuation---------------------
neck_joint_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: neck_joint
    joint_speed: 2.0
    motor:
        id: 1
        init: 630
        min: 300
        max: 970
#------------------right arm actuation---------------------
right_shoulder_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_shoulder_roll_joint
    joint_speed: 2.0
    motor:
        id: 2
        init: 720
        min: 520
        max: 1023

right_shoulder_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_shoulder_pitch_joint
    joint_speed: 2.0
    motor:
        id: 3
        init: 300
        min: 0
        max: 480

right_elbow_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_elbow_pitch_joint
    joint_speed: 2.0
    motor:
        id: 4
        init: 300
        min: 0
        max: 480

#------------------left arm actuation---------------------
left_shoulder_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_shoulder_roll_joint
    joint_speed: 2.0
    motor:
        id: 5
        init: 780
        min: 390
        max: 800

left_shoulder_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_shoulder_pitch_joint
    joint_speed: 2.0
    motor:
        id: 6
        init: 480
        min: 300
        max: 1023

left_elbow_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_elbow_pitch_joint
    joint_speed: 2.0
    motor:
        id: 7
        init: 650
        min: 270
        max: 1023
